fix: LEAP-314: Fix Github vulnerabilities

robot-ci-heartex opened the pull request:
Hi @juliosgarbi!

This PR was [created](https://github.com/HumanSignal/label-studio/actions/runs/6708195590) in a response to PRs in upstream repos:
- https://github.com/HumanSignal/label-studio-frontend/pull/1609

juliosgarbi committed:
[submodules] Build static HumanSignal/label-studio-frontend

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/6708195590

yyassi-heartex reviewed (changes requested):
taking back my approval of this PR - i only see the version.json file here - i expect to see updated build files too

juliosgarbi committed:
[submodules] Build static HumanSignal/label-studio-frontend

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/6709547772

juliosgarbi committed:
Merge branch 'origin/develop' into 'fb-leap-314'
